apps/root/src/common/components/token-selector/index.tsx

```diff
@@ -112,31 +129,38 @@ const TokenSelector = ({ handleChange, selectedToken }: TokenSelectorProps) => {
     [balances]
   );
 
-  const items: OptionWithKeyAndToken[] = React.useMemo(
-    () =>
-      orderBy(
-        [
-          ...availableTokens,
-          ...(customToken
-            ? [
-                {
-                  ...customToken,
-                  key: customToken.token.address,
-                },
-              ]
-            : []),
-        ],
-        [({ balanceUsd, token }) => parseFloat(formatUnits(balanceUsd || 0n, token.decimals + 18))],
-        ['desc']
-      ),
-    [availableTokens, customToken]
-  );
+  const items: OptionWithKeyAndToken[] = React.useMemo(() => {
+    const parsedWithCustomTokens = availableTokens.map((tokenOption) => ({
+      ...tokenOption,
+      token: {
+        ...tokenOption.token,
+        isCustomToken:
```

Comment:
Carefull with this, if we later add that custom token to our list, this is still gonna show as a custom token since it would still be maintained in their local storage, when it is already curated by us

Reply:
I've included a solution for this, but I'm thinking if we should have a cleanCustomToken action, like we have for transactions